### DISCUSSION TITLE
testing

### DISCUSSION THREAD

**Discussion by LexterS999** [TechnitiumSoftware/DnsServer#291] [General]
How can I install testing version placed in development stage?

**Comment by ShreyasZare** [comment]
Thanks for asking. There is no dev version made available as an installer. You can however clone the repo (plus the TechnitiumLibrary repo) and run it from the develop branch. All you will need is to have visual studio 2019 installed. There is no guarantee made that the code in develop branch will be in a working condition though since a lot of things are pushed weekly.

**Comment by LexterS999** [comment]
So there is no other way?

**Reply by ShreyasZare** [reply]
No since there is no beta releases maintained. Is there any specific thing you are looking at? Do let me know.
